tests/validator_test.py

```diff
@@ -0,0 +1,47 @@
+'''
+Series of tests to ensure that new code added to the xml validator and the associated xml to html transforms actually works as expected. This will also begin to put in place the test suite for the validator and associated transforms.
+'''
+import unittest
+
+
+# Start with importing the necessary bits - what do I need to import? I think I need to bring in the entire utils.py bit and maybe the cnxml to html bit because these are what I wish to test. I may also need some stuff from the monassis.buildout repo such as the core.py file in transforms
```

Comment:
Some very long lines and comments in this file - please reformat them to fit inside of the 100 character mark.